SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -357,13 +357,13 @@ HeroFusion = 1
 # Волшебное Зеркало работает и против войск ИИ игрока
 # Der Magische Spiegel des Spielers ist berücksichtigt, wenn die künstliche Intelligenz einen Zauberspruch wirkt
 # Το ξόρκι {Magic Mirror (Μαγικός Kαθρέφτης)} λαμβάνεται υπ' όψιν όταν η Τεχνητή Νοημοσύνη χρησιμοποιεί ένα ξόρκι
-AIMagicMirror = 1
+AIMagicMirror = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.